setup.py

```diff
@@ -27,8 +27,12 @@
 #------------------------------------------------------------------------------
 
 from setuptools import setup
+import re
 
-import adal
+# setup.py shall not import adal
+__version__ = re.search(
+    "__version__[\s]*=[\s]*(.*)[\s]*",
```

Comment:
Up to you, but i think since we are using a regular expression, we might better off be more accurate like requests package's [logic](https://github.com/kennethreitz/requests/blob/master/setup.py#L51). Their logic will handle trailing comments, multiple line (rare though), etc